icu4c/source/i18n/number_formatimpl.cpp

```diff
@@ -481,8 +481,10 @@ NumberFormatterImpl::macrosToMicroGenerator(const MacroProps& macros, bool safe,
 }
 
 const PluralRules*
-NumberFormatterImpl::resolvePluralRules(const PluralRules* rulesPtr, const Locale& locale,
-                                        UErrorCode& status) {
+NumberFormatterImpl::resolvePluralRules(
```

Comment:
optional: overactive auto-formatting here & there?

Reply:
I noticed this too-- it'd be nice if things that are purely formatting changes were in their own commit.

Reply:
At one point I had intended to change the arguments to this function but then decided not to. So I can just revert this change

Reply:
I didn't end up reverting this; ok to keep the formatting change?